config/initializers/redis.rb

```diff
@@ -0,0 +1,7 @@
+require 'connection_pool'
+
+redis_url = "#{ENV['REDIS_QUEUE_URL']}/0"
```

Comment:
This probably needs to go into the other Redis instance (at `ENV['REDIS_CACHE_URL]`). I'd probably name the initialiser something like `redis_objects.rb`, as we're using Redis in a bunch of places

Reply:
Good spot, I've changed it to the Redis cache and renamed the initialiser

Reply:
I wonder if there is advantage to be had by further separating this type of caching by using a different database eg. `/1`

Thinking aloud:

- we'd be able to connect to each redis database separately to interrogate the contents for debugging, we'd also be able to flush and delete keys with less risk to dumping the geolocation cache
- there are likely performance advantages for reads in redis when it doesn't have to go through +20k geolocation records to find the page view ones, and vice versa!

Reply:
I did wonder this. Worth trying in dev I reckon, but I can't see why it wouldn't work

Reply:
Or do we want to have a more sensible naming convention for the Redis databases? So `/geolocation` and `/pageviews` for example

Reply:
Can we do meaningful names? I assumed from past examples and memory it was only integers!

Reply:
No, you're right, it is only integers. I guess we could refer to them by name internally, but I dunno if that would be that valuable